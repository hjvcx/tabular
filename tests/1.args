tabular -i ';' -n '^' -f r1l2l4r5 -o a persons.csv persons.tab