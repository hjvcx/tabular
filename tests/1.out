  |       |         |        |    
--+-------+---------+--------+----
9 |Николай|Иванович |Синичкин|71,6
10|Пётр   |Андреевич|Орлов   |28,4
